core/src/main/scala/cats/data/EitherT.scala

```diff
@@ -1187,13 +1187,13 @@ private[data] trait EitherTSemigroupK[F[_], L] extends SemigroupK[EitherT[F, L,
   implicit val F: Monad[F]
   def combineK[A](x: EitherT[F, L, A], y: EitherT[F, L, A]): EitherT[F, L, A] =
     EitherT(F.flatMap(x.value) {
-      case l @ Left(_)  => y.value
+      case _ @Left(_)   => y.value
       case r @ Right(_) => F.pure(r)
```

Comment:
I think that since the value of `Left` is not used here, then it would be more efficient to change the order of matching:
```suggestion
      case r @ Right(_) => F.pure(r)
      case _            => y.value
```
And the same is applicable to the partial function below.

Reply:
Sounds good to me.